### PR TITLE
Ihodge volunteer network logic

### PR DESCRIPTION
**Pull request by ihodge97**

### CODE REVIEW

File: volunta/screens/ProfileScreen.js [hunk 1]
```diff
@@ -54,24 +54,18 @@ export default class ProfileScreen extends React.Component {
...
-            await getUsersAttributes(snapshot.docs, ['name', 'profile_pic_url'])
-        ),
       getUserInterestNames(userId),
     ]);
 
     // get name of user
     const profileName = await getProfileName(userId);
 
+    // get the volunteer network from the past events
+    const volunteerNetwork = await this._getVolunteerNetwork(pastEvents);
```
Comment (susannahameyer) [inline volunta/screens/ProfileScreen.js]: Can move this into the Promise.all call but we can also do a round of consolidating after

File: volunta/screens/ProfileScreen.js [hunk 2]
```diff
@@ -98,6 +92,30 @@ export default class ProfileScreen extends React.Component {
     });
   };
 
+  //Gets the volunteer network of this user based on their past service events
+  _getVolunteerNetwork = async pastEvents => {
```
Comment (susannahameyer) [inline volunta/screens/ProfileScreen.js]: should we keep in api.js for consistency?

Comment (lfvarela) [reply volunta/screens/ProfileScreen.js]: yep

File: volunta/screens/ProfileScreen.js (hunk 1 above)
Comment (lfvarela) [reply volunta/screens/ProfileScreen.js]: agreed, should speed things up a bit right?

**Pull request merged**